Source/player.h

```diff
@@ -400,6 +401,9 @@ struct Player {
 
 	bool CanUseItem(const Item &item) const
 	{
+		if (!IsItemValid(*this, item))

```

Comment:
This doesn't compile...

Reply:
That's the current problem I've been trying to fix